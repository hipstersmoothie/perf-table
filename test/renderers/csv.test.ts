import renderCSV from '../../src/renderers/csv';

const data = [
  ['name', 'age', 'pizza topping'],
  ['burt', '40', 'anchovies'],
  ['gert', '35', 'pineapple'],
  ['john', '18', 'basil']
];

test('renderCSV', () => {
  expect(renderCSV(data)).toMatchSnapshot();
});
