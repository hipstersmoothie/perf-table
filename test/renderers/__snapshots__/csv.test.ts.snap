// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderCSV 1`] = `
"
name,age,pizza topping
burt,40,anchovies
gert,35,pineapple
john,18,basil"
`;
